File: app/src/main/java/org/metabrainz/mobile/data/sources/api/entities/mbentity/Work.java
package org.metabrainz.mobile.data.sources.api.entities.mbentity;

class Work extends MBEntity {
    private String title;
    //TODO: Implement Relations

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    @Override
    public String toString() {
        return "Work{" +
                "title='" + title + '\'' +
                ", mbid='" + mbid + '\'' +
                ", disambiguation='" + disambiguation + '\'' +
                '}';
    }
}

File: app/src/main/java/org/metabrainz/android/data/sources/api/entities/mbentity/Work.kt
package org.metabrainz.android.data.sources.api.entities.mbentity

internal class Work : MBEntity() {
    //TODO: Implement Relations
    var title: String? = null
    override fun toString(): String {
        return "Work{" +
                "title='" + title + '\'' +
                ", mbid='" + mbid + '\'' +
                ", disambiguation='" + disambiguation + '\'' +
                '}'
    }
}